tests/src/FunctionalJavascript/CorporateSiteInfoSettingsFormTest.php

```diff
@@ -48,8 +48,15 @@ public function testConfigurationForm(): void {
     $page = $this->getSession()->getPage();
     $assert_session = $this->assertSession();
 
+    // Site owner and Content owners field is required.
+    $page->pressButton('Save configuration');
+    $this->assertTrue($this->getSession()->evaluateScript("return jQuery(document).find('#edit-site-owner:invalid')"));
```

Comment:
Let's use the typical approach of disabling the required field browser validation: see `disableNativeBrowserRequiredFieldValidation()` in multiple components.